example/protobuf-sync-message/provider/composer.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "pact-foundation/example-protobuf-sync-message-provider",
+    "require": {
+        "grpc/grpc": "^1.57",
+        "spiral/roadrunner-grpc": "^2.0",
+        "tienvx/composer-downloads-plugin": "^1.2"
+    },
+    "require-dev": {
+        "ext-grpc": "*"
```

Comment:
This line should be in `require`, not `require-dev`. But if it is in `require`, `php-cs` and compatibility suite jobs will need to install `grpc` extension, which make them slow significantly

Reply:
What about using `"suggest"` in  composer.json?

```
"suggest": {
    "ext-grpc": "If you want to use gRPC then you must install and use this"
}
```

Reply:
Updated.

The suggestion is hidden in this message:

```
18 package suggestions were added by new dependencies, use `composer suggest` to see details.
```

I have to run this command  `composer suggest` to see it clearly. But it's better than nothing.

Reply:
Sorry, I think I should have been clearer.

The package should exist within `require-dev` as well as `suggest`:

- Suggest presence is to let devs know that if they want gRPC they should install the extension
- Require-dev presence is to clearly communicate that to run the test suite, or work on this project, because tests contain gRPC, they will need to find a way to install that package.

Reply:
Reverted. Now **this** `composer.json` contains both `require-dev` and `suggest`. I'm not sure which is better:  move `suggest` to the **root** `composer.json`, or keep it here.